client-js/app/scripts/registry.ts

```diff
@@ -9,182 +11,10 @@ export interface RegistryListing {
   vals: Array<string>;
 }
 
-export interface RegistryApi {
```

Comment:
Yup, I didn't understand what was specific to the old http RPC protocol and what was not. I just kept hacking away everthing I could that kept the app still running.